src/components/GoodList/GoodList.tsx

```diff
@@ -0,0 +1,13 @@
+import { GoodCard } from '../GoodCard/GoodCard';
+
+interface List {
+  goods: string[];
+}
+
+export const GoodList: React.FC<List> = ({ goods }) => (
+  <ul>
+    {goods.map((good, index) => (
+      <GoodCard good={good} key={`id:${index}`} />
```

Comment:
Using the array index as a key in React is not recommended if the list can change, as it may negatively affect performance and component state. If each `good` has a unique identifier, use that as the key instead. [CODE KNOWLEDGE]